src/vector/jitsi/index.html

```diff
@@ -10,7 +10,6 @@
     <div class="joinConferenceFloating">
         <div class="joinConferencePrompt">
             <span class="icon"><!-- managed by CSS --></span>
-            <!-- TODO: i18n -->
```

Comment:
The h2 still isn't translated though?

Reply:
It is:

https://github.com/vector-im/element-web/blob/d9431e0d5956be978aafdfb632d4896a3ae16a42/src/vector/jitsi/index.ts#L235

I can't do it inline, but I'm applying it later as soon as a I know the current language.

Reply:
a comment to indicate that would be appreciated :)